devito/dle/transformer.py

```diff
@@ -447,3 +555,23 @@ def _summary(self, mode):
             steps = " --> ".join("(%s)" % i for i in self.timings.keys())
             elapsed = sum(self.timings.values())
             dle("%s [%.2f s]" % (steps, elapsed))
+
+
+# Utilities
+
+"""
+A dictionary to quickly access standard OpenMP pragmas
+"""
+omplang = {
+    'par-region': c.Pragma('omp parallel'),
+    'par-for': c.Pragma('omp parallel for schedule(static)'),
+    'par-for-collapse2': c.Pragma('omp parallel for collapse(2) schedule(static)'),
+    'simd-for': c.Pragma('omp simd')
+}
+
+"""
+Compiler-specific language
+"""
+complang_ALL = {
+    'IntelCompiler': {'ignore-deps': c.Pragma('ivdep')}
```

Comment:
that's actually the reason I preferred to have a separate dict. In theory, the dict here has everything concerning the language, while the dict in ``IntelCompiler`` contains all of the compilation flags. So, slightly different distinction, but debatable